metaflow/plugins/argo/argo_live_run.py

```diff
@@ -82,60 +80,69 @@ def trigger(
             time (in minutes) to wait for run to finish
         """
 
+        template_name = kwargs.get("template_name")
+        live_run = cls(flow_name=flow_name, template_name=template_name)
+
         if parameters is None:
             parameters = {}
 
         # convert dicts/lists to json-valid strings
         for key in parameters:
-            if isinstance(parameters[key], (dict, list)):  # TODO: discuss best way to filter
+            if isinstance(
+                parameters[key], (dict, list)
+            ):  # TODO: discuss best way to filter
                 try:
                     parameters[key] = json.dumps(parameters[key])
-                except:
-                    raise TypeError(f"Parameter with key '{key}' not supported.\n"
-                                    "Supported types are str and json-convertible objects")
+                except TypeError:
+                    raise TypeError(
+                        f"Parameter with key '{key}' not supported.\n"
+                        "Supported types are str and json-convertible objects"
+                    )
```

Comment:
Please also test how non JsonType parameters in some workflow.

Reply:
Note: clarified via slack and added test cases